sigrokdecode/__init__.py

```diff
@@ -106,16 +115,20 @@ def wait(self, conds=[]):
                 data_conds.append(data_cond)
 
         raw_data = self.input.wait(data_conds)
-        data = [None] * (len(type(self).channels) + len(getattr(type(self), "optional_channels", [])))
+        data = [None] * (
+            len(type(self).channels) + len(getattr(type(self), "optional_channels", []))
+        )
         for decoder_channel in self.decoder_channel_to_data_channel:
             data_channel = self.decoder_channel_to_data_channel[decoder_channel]
             data[decoder_channel] = raw_data[data_channel]
 
         return tuple(data)
 
-    def put(self, startsample: int, endsample: int, output_id: OutputType, data: DataType) -> None:
+    def put(
+        self, startsample: int, endsample: int, output_id: OutputType, data: DataType
```

Comment:
output_id is OutputType in pysigrok but in normal sigrok I think its just an int. Here it is meant to be opaque really.